test/integration/browser_screenshot_test.rb

```diff
@@ -60,20 +60,12 @@ def test_screenshot_with_blur_active_element_disabled
 
   def test_screenshot_selected_element
     visit "/"
-    screenshot "form_element", crop: bounds('form')
+
+    screenshot "cropped_screenshot", crop: [0, 100, 100, 200]
   end
 
   private
 
-  def bounds(selector)
-    element = evaluate_script("document.querySelector('#{selector}').getBoundingClientRect()")
-    if element['left']
-      [element['left'], element['top'], element['right'], element['bottom']]
-    else
-      [0, 0, *window_size]
-    end
-  end
```

Comment:
yep, that's what I'm thinking will be great to have. But for simplicity and to make it more explicit let's avoid it. And adds such utils as lib in the #46